internal/exec/go_getter_utils.go

```
@@ -8,9 +8,11 @@
 	"net/url"
 	"os"
 	"path/filepath"
+	"regexp"
 	"strings"
 	"time"
 
+	"github.com/charmbracelet/log"
```

golangci-lint (1.62.2)

import "github.com/charmbracelet/log" imported without alias but must be with alias "log" according to config

(importas)